modules/shadertools/src/modules/phong-lighting/phong-lighting.glsl.js

```diff
@@ -29,6 +29,8 @@ uniform float lighting_diffuse;
 uniform float lighting_shininess;
 uniform vec3  lighting_specularColor;
 
+uniform bool lighting_enabled;
```

Comment:
Nit: Most shader modules have a uniform naming convention using a leading `u` like `lighting_uEnabled`?

Reply:
thanks for reminding, will fix them